fvtest/jitbuildertest/ConvertBitsTest.cpp

```diff
@@ -105,8 +105,8 @@ TEST_F(BitConversionTest, Double2Long)
    BitConvertDouble2LongFunction* double2long;
    ASSERT_COMPILE(TR::TypeDictionary, BitConvertDouble2Long, double2long);
 
-   ASSERT_EQ(0, double2long(0.0));
+   ASSERT_EQ(UINT64_C(0), double2long(0.0));
    ASSERT_EQ(0xbff0000000000000ULL, double2long(-1.0));
```

Comment:
My initial thought was, "because 0xbff0000000000000 is already long enough".  But here we are getting into that area that C99 (or whatever variant) denies any reasoning; things like, what if 0xbff0000000000000 doesn't fit in ULL?  In practice, the scope of this PR is to compile and exhibit the expected behaviour on those platforms we have or can reasonably expect to have in the foreseeable future, in order to unblock work on the master branch.  This PR satisfies our "standard" platforms and I tested on two extra platforms (arm and the recently-discussed e500); beyond that, as a person working in the area of formal verification, I am rather skeptical of the usefulness of more legalistic analysis of the standard than what we've done here.

Reply:
> "because 0xbff0000000000000 is already long enough"

I can understand that. However, my concern is not about correctness but about consistency (I am fairly certain this code will work on any platform this project cares about). In my experience, inconsistent code is confusing; more confusing than long code. If others see no value in this, I can accept the code as is.

Reply:
Looks like no-one else has expressed an opinion on this in the past 8 days; shall we move on?  In my view we have reached the point of diminishing returns, I can happily live either with that UINT64_C or without.

Reply:
Hey sorry I was waiting for your reply and you were waiting for us :)... Can you make the change to UINT64_C and I will get it merged today?